client/src/outside/components/AssignmentsDialog.vue

```diff
@@ -40,32 +62,74 @@ const ERRORS = {
   NAME_REQUIRED: 'Assignment name is required',
   ID_REQUIRED: 'Assignment externalId is required',
   ID_PATTERN: 'External Id needs to contain letters, numbers, hypens or underscores',
-  SEED_REQUIRED: 'Assignment EHR data seed is required'
+  SEED_REQUIRED: 'Assignment EHR data seed is required',
+  INVALID_TIME: 'Please, enter a valid 24hrs time'
 }
 
 const EDIT_ACTION= 'edit'
 const CREATE_ACTION = 'create'
 
+const PERSONA_DEFAULT_VALUES = {
```

Comment:
Apologies for the misunderstanding and for the second thoughts about defaults.  Originally, I thought we should provide defaults **on the EHR side ** for legacy assignments. Now, I think we need another approach entirely.  So, for the time being please remove the defaults.
Also, make these four values optional. If present then, yes please, check that the time and day values are valid.